packit/config/job_config.py

```diff
@@ -65,10 +69,16 @@ def __init__(
             preserve_project: if set, project will not be created as temporary
             additional_packages: buildroot packages for the chroot [DOES NOT WORK YET]
             additional_repos: buildroot additional additional_repos
-            use_internal_tf: if we want to use internal instance for Testing Farm
+            fmf_url: - git repository containing the metadata (FMF) tree
+            fmf_ref: - branch, tag or commit specifying the desired git revision
+            use_internal_tf: if we want to use internal instance of Testing Farm
             skip_build: if we want to skip build phase for Testing Farm job
         """
-        self.targets: Set[str] = set(targets) if targets else set()
+        self._targets: Dict[str, Dict[str, Any]]
```

Comment:
Can we use `Union` here?

Do I get it right that instead of `Any`, we can use `List[str]`?

```suggestion
        self._targets: Dict[str, Dict[str, List[str]]]
```

Reply:
> Can we use `Union` here?

No, the `__init__` argument `_targets` is `Union`, because it can be either a list or a dict.
But `JobMetadataConfig` attribute `_targets` is a dict only.

> Do I get it right that instead of `Any`, we can use `List[str]`?

No, the `distros` value is a list, but there might later be other keys whose values might be something else (than a list).

Reply:
OK, the forward compatibility. No problem then.